src/sagemaker/task.py

```diff
@@ -0,0 +1,49 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Accessors to retrieve task fallback input/output schema"""
+from __future__ import absolute_import
+
+import json
+import os
+from typing import Any, Tuple
+
+
+def retrieve_local_schemas(task: str) -> Tuple[Any, Any]:
```

Comment:
I think this should be in serve/utils https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/serve/utils